PEC_VerificationCodeMessage/index.mjml

```diff
@@ -0,0 +1,92 @@
+    <mj-title>Il tuo codice di verifica</mj-title>
+    <mj-font
+      name="Titillium Web"
+      href="https://selfcare.pagopa.it/assets/font/selfhostedfonts.css"
+    />
+    <mj-attributes>
+      <mj-all font-family="Titillium Web, system-ui, sans-serif" />
+    </mj-attributes>
+    <mj-raw>
+      <meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
+      <meta name="color-scheme" content="light dark" />
+      <meta name="supported-color-schemes" content="light dark" />
+    </mj-raw>
+  </mj-head>
+  <mj-body css-class="mail-body">
+    <mj-include path="../style.css" type="css" />
+    <mj-wrapper>
+      <!-- PagoPA Logo -->
+      <mj-raw>
+        <div class="text" style="text-align: left; margin-bottom: 32px;">
+          <img alt="SEND - Servizio Notifiche Digitali" src="data:image/png;base64, ${logoBase64}" style="display: block;width: 116px;border: 0;height: auto;line-height: 100%;outline: none;text-decoration: none;-ms-interpolation-mode: bicubic;padding: 10px 25px"
+          width="116">
+        </div>
+      </mj-raw>
+
+      <mj-section css-class="section" padding-bottom="0px" padding-top="0px">
+        <mj-column>
+          <!-- Title -->
+          <mj-text
+            css-class="text"
+            font-weight="700"
+            align="left"
+            padding-top="16px"
+            padding-bottom="16px"
+            color="#17324D"
+            font-size="32px"
+            line-height="40px"
+          >
+            <!-- H1 Desktop (from MUI Italia)-->
+            <h1 style="font-size: 32px; font-weight: 700; mso-line-height-rule: exactly; line-height: 40px; margin: 0;">
+              Il tuo codice di conferma
+            </h1>
+          </mj-text>
+
+          <!-- Main Message -->
+          <mj-text
+            css-class="text"
+            font-size="18px"
+            line-height="28px"
+            font-weight="400"
+            align="left"
+            padding-top="16px"
+            padding-bottom="0px"
+            color="#17324D"
+          >
+            <p style="margin-top: 0px">Ciao,</p>
+          </mj-text>
+          <mj-text
+            css-class="text"
+            font-size="18px"
+            line-height="28px"
+            font-weight="400"
+            align="left"
+            padding-top="0px"
+            color="#17324D"
+          >
+            <p style="margin-top: 0px; font-size: 18px; mso-line-height-rule: exactly; line-height:28px">
+              Per confermare questo indirizzo PEC come recapito legale, inserisci su SEND il codice</p>
+              <h5 style="font-size: 28px; line-heigth:42px; margin-top:0">${verificationCode}</h5>
+          </mj-text>
+          <mj-text
+            css-class="text"
+            font-size="14px"
+            line-height="20px"
+            padding-bottom="0px"
+            color="#17324D"
+          >
+            <p style="margin-top: 0px">
+              A presto,<br aria-hidden="true" />Piattaforma Notifiche
```

Comment:
@SarahDonvito copy is wrong, this should be "A presto, SEND - Servizio Notifiche Digitali"